scripts/travis/deploy_cli_macos.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
```

Comment:
Add environment variable check like

https://github.com/sql-machine-learning/sqlflow/blob/0afb60abc58e8f42fe151a18b990230830a1c76c/scripts/travis/deploy.sh#L34-L37

So that normal PR won't fail.

Reply:
Done.